tests/foreman/api/test_organization.py

```diff
@@ -349,9 +349,9 @@ def test_positive_add_smart_proxy(self):
         @CaseLevel: Integration
         """
         # Every Satellite has a built-in smart proxy, so let's find it
-        smart_proxies = entities.SmartProxy().search()
-        self.assertGreater(len(smart_proxies), 0)
```

Comment:
Another benefit of using assertion here is the fact it will result in test marked with `fail`, not `error`. In some meaning it's debatable too - in case we've lost default capsule it's not test's fault imo, but let's better stick to failures instead of errors.